packages/core-js/internals/object-create.js

```diff
@@ -26,13 +37,27 @@ var NullProtoObject = function () {
   iframe.src = String(JS);
   iframeDocument = iframe.contentWindow.document;
   iframeDocument.open();
-  iframeDocument.write(LT + SCRIPT + GT + 'document.F=Object' + LT + '/' + SCRIPT + GT);
+  iframeDocument.write(scriptTag('document.F=Object'));
   iframeDocument.close();
-  NullProtoObject = iframeDocument.F;
-  while (length--) delete NullProtoObject[PROTOTYPE][enumBugKeys[length]];
-  return NullProtoObject();
+  return iframeDocument.F;
 };
 
+// Check for document.domain and active x support
+// No need to use active x approach when document.domain is not set
+// see https://github.com/es-shims/es5-shim/issues/150
+// variation of https://github.com/kitcambridge/es5-shim/commit/4f738ac066346
+var activeXDocument;
```

Comment:
Seems it's not required in this scope.

Reply:
When declared inside NullProtoObject, IE8 will throw an error, which message is around "Unable to run released script". So I have to declared globally

Use activeXDocument inside NullProtoObjectViaActiveX, rather than passing through parameters, should have the same problem.